pkg/web/templates/partials/counterparty/counterparty_detail.html

```diff
@@ -1 +1,62 @@
-<div></div>
\ No newline at end of file
+{{ with .CounterpartyDetail }}
+<div class="modal-box max-w-3xl">
+  <div class="mb-4 flex justify-between items-center">
+    <h1 class="font-bold text-xl">Counterparty Details</h1>
+    <button onclick="cparty_modal.close()" class="btn btn-sm btn-circle btn-ghost">
+      <i class="fa-solid fa-x"></i>
+      <span class="sr-only">Close modal</span>
+    </button>
+  </div>
+  <dl>
```

Comment:
Should we include the counterparty's IVMS Record here?

Reply:
Not just yet, let's wait until we get to the facelift to start doing heavy duty data.